feathr_project/feathr/_envvariableutil.py

```diff
@@ -59,5 +61,8 @@ def get_environment_variable(variable_key):
         password = os.environ.get(variable_key)
         if not password:
             logger.info(variable_key +
-                        ' is not set in the environment variables.')
+                        ' is not set in the environment variables. Try Key Vault value.')
+            akv_name = os.env.get("KEY_VAULT__NAME")
```

Comment:
[nitpick] Suggest changing it to KEY_VAULT_NAME , same number of underscores.

Reply:
was intended to fit the naming pattern of yaml config attributes. But agree, we can use KEY_VAULT_NAME for current functions